### PR TITLE
Add pipelines to census example

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

File: sklearn/notebooks/Online Prediction with scikit-learn.ipynb [hunk 1]
```diff
@@ -207,37 +206,80 @@
...
+    "    if col in CATEGORICAL_COLUMNS:\n",
+    "        # Create a scores array to get the individual categorical column\n",
+    "        # Example:\n",
+    "        # data = [39, 'State-gov', 77516, 'Bachelors', 13, 'Never-married', 'Adm-clerical', \n",
+    "        #    'Not-in-family', 'White', 'Male', 2174, 0, 40, 'United-States']\n",
+    "        # scores = [0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]\n",
+    "        #\n",
+    "        # Returns: [['Sate-gov']]\n",
+    "        scores = []\n",
+    "        for j, col2 in enumerate(COLUMNS[:-1]):\n",
```
Comment (sirtorry) [inline sklearn/notebooks/Online Prediction with scikit-learn.ipynb]: i, j, col, col2 are slightly confusing variable names

Comment (nnegrey) [reply sklearn/notebooks/Online Prediction with scikit-learn.ipynb]: I can switch that line to ` for j in range(len(COLUMNS[:-1])):`

i and j are indexes. 
col is the column

Comment (sirtorry) [reply sklearn/notebooks/Online Prediction with scikit-learn.ipynb]: actually removing col2 would make it clearer.

my confusion largely arose from *visualising* what the indexes are doing. the comments above the loops help but not immediately clear that we're creating the 1 hot array by simply comparing the column indexes.

**Comment by nnegrey** [comment]
Could make this file a 2nd one named `Online Prediction with Pipelines` (and keep this simple version as well, add a README to clarify simple vs. advanced solution)
@sirtorry 
@dizcology 
@happyhuman 
Thoughts?

**Comment by thedriftofwords** [comment]
+1 on keeping the previous notebook. I'm keeping the previous content in the docs.

**Pull request merged**